api/v1/db/oracledb/game-dao.js

```diff
@@ -195,6 +224,48 @@ const deleteGameByGameId = async (gameId) => {
   }
 };
 
+const databaseName = (string) => {
+  if (string === 'round') {
+    return 'ROUND_ID';
+  }
+  return decamelize(string).toUpperCase();
+};
+
+const isTruthyOrZero = val => (val || val === 0);
+
+const patchGame = async (gameId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const { tableCards } = attributes;
+    delete attributes.tableCards;
+    attributes.round = attributes.round ? attributes.round[0].toUpperCase() : null;
+    if (!_.isEmpty(tableCards)) {
+      await cleanTableCardsByGameId(gameId, connection);
+      await insertCardsByGameId(gameId, tableCards, connection);
+    }
+    const filteredAttributes = _.pickBy(attributes, isTruthyOrZero);
+    const joinedStringArray = _.map(filteredAttributes,
+      (value, key) => (`${isTruthyOrZero(value) ? `${databaseName(key)} = :${key}` : ''}`));
+    const joinedString = _(joinedStringArray).compact().join(', ');
+    const patchSqlQuery = `
+    UPDATE GAMES
+    SET ${joinedString}
+    WHERE GAME_ID = :id
+    `;
+    if (_.isEmpty(filteredAttributes)) {
+      await connection.commit();
+      return true;
+    }
```

Comment:
This check that `filteredAttributes` is empty should probably go directly after it's defined, since at that point, you'll know if you need to return early.